### PR TITLE
Remove unused `static/tests`

### PR DESCRIPTION
**Pull request by pedropombeiro**
This PR removes the unused tests in `static/tests` and moves `_assets/package.json` to `_assets/static/package.json`, since that file is now only used by make generate.

>This PR updates the `npm test` script after changes in the `_assets` refactor and migration to Whisper v6.
>
>The main idea is that it'll create a symbolic link to the `_assets/node_modules` folder in the project root, and delete that after the tests. That allows the tests in `static/tests` to find the node modules. It also increases the default timeout for tests so that they have time to wait for the mailserver (normally 10s). The tests still don't pass completely, but I'm not sure if they were passing previously. At least we have a working base now.

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]
Awesome 🌟

**Comment by JekaMas** [comment]
Does js tests more useful for us? For me, we should have only golang tests to run them with '-race'.

**Comment by pedropombeiro** [comment]
No idea why the tests are written in js, but if we want to port them to Go, that would be better done as a bounty.

**Comment by adambabik** [comment]
There are not used and are outdated. We can safely remove them.

**Comment by pedropombeiro** [comment]
OK, I'll change PR to be about removing them. That way we avoid any future confusion about these tests.

**Comment by pedropombeiro** [comment]
@adambabik can you please take a second look? I moved `_assets/package.json` to `_assets/static/package.json`, since now that file is only used by `make generate`.

File: Makefile [hunk 1]
```diff
@@ -102,7 +102,8 @@ xgo:
 	go get github.com/karalabe/xgo
 
 generate: ##@other Regenerate assets and other auto-generated stuff
-	cp ./_assets/node_modules/web3/dist/web3.min.js ./static/scripts/web3.js
+	(cd _assets/static && npm install)
```
Comment (adambabik) [inline Makefile]: Why is it wrapped in parentheses?

**Review by adambabik — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (pedropombeiro) [reply Makefile]: you're right, that's not needed here. I'll remove it.

Comment (pedropombeiro) [reply Makefile]: Other than that, does it look OK?

**Pull request merged**